package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "esn-frontend-group",
+  "version": "1.9.0-dev",
+  "description": "OpenPaaS module for group feature",
+  "main": "index.js",
+  "devDependencies": {
+    "angular": "1.3.20",
+    "angular-mocks": "1.3.20",
+    "chai": "3.0.0",
+    "chai-datetime": "1.4.0",
+    "chai-shallow-deep-equal": "1.4.0",
+    "css-loader": "^3.6.0",
+    "file-loader": "^6.0.0",
+    "ignore-loader": "^0.1.2",
+    "karma": "4.2.0",
+    "karma-mocha": "1.1.1",
+    "karma-firefox-launcher": "1.1.0",
+    "karma-sinon-chai": "^2.0.2",
+    "karma-spec-reporter": "0.0.26",
+    "karma-webpack": "^4.0.2",
+    "less-loader": "^6.1.1",
+    "mocha": "6.2.0",
+    "pug": "^3.0.0",
+    "pug-loader": "^2.4.0",
+    "sinon": "1.17.5",
+    "sinon-chai": "2.8.0",
+    "style-loader": "^1.2.1",
+    "svg-inline-loader": "^0.8.2",
+    "webpack": "^4.43.0",
+    "webpack-cli": "^3.3.11"
+  },
+  "scripts": {
+    "serve": "webpack-dev-server --watch --progress",
```

Comment:
Do we need this if this repo is not an SPA?

Reply:
yep sorry that slipped in, used it to test some stuff